server/api/controllers/asgs/asgController.js

```diff
@@ -12,7 +14,7 @@ let SetAutoScalingGroupSize = require('commands/asg/SetAutoScalingGroupSize');
 let SetAutoScalingGroupSchedule = require('commands/asg/SetAutoScalingGroupSchedule');
 let UpdateAutoScalingGroup = require('commands/asg/UpdateAutoScalingGroup');
 let GetAutoScalingGroupScheduledActions = require('queryHandlers/GetAutoScalingGroupScheduledActions');
-let Environment = require('models/Environment');
```

Comment:
`Environment` is still used by `asgController/getAsgByName`, resulting in unhandled promise rejection for `/api/v1/asgs/{name}?environment={environment}`